mkt/settings.py

```diff
@@ -94,6 +94,7 @@
 DATABASES['default']['OPTIONS'] = {'init_command': 'SET storage_engine=InnoDB'}
 DATABASES['default']['TEST_CHARSET'] = 'utf8'
 DATABASES['default']['TEST_COLLATION'] = 'utf8_general_ci'
+DATABASES['default']['ATOMIC_REQUEST'] = True
```

Comment:
There and in all settings, it should be `ATOMIC_REQUEST*S*`.

Which means we need a test proving that the views are correctly wrapped into a transaction, since tests were passing despite this typo.